README.md

```diff
@@ -28,10 +28,82 @@ Please see [ember-flight-icons/README](ember-flight-icons/README.md). This addon
 
 Please see [flight-icons/README](flight-icons/README.md). This addon can be used in React or Ember apps.
 
+### How to consume an icon in Figma
+
+1. Open your project file within Figma
+2. Navigate to the Assets panel in the left sidebar
+3. Enable the [Flight Icons](https://www.figma.com/file/TLnoT5AYQfy3tZ0H68BgOr/Flight-Icons?node-id=164%3A0) Library  
+_Click the Team library icon (looks like an open book), toggle on Flight Icons_
+4. Search or scroll to find the icon you want to use and drag it into your file
+
+💡 Tip: Swapping instances, sizes, and colors is easy to do from the right sidebar
+
+## Best Practices
+
+### Icon Types
+
+![example of icon types](ember-flight-icons/tests/dummy/public/assets/images/icon-types.png)
+
+* Use the `Generic` style icons by default
+* Use the `Filled` style if indicating the toggled state of an icon
+* Use the `Filled` style where contrast against other icons is important
+* Use the `Off` style icons to indicate a disabled state
+
+### Icon Sizes
+
+![example of icon sizes](ember-flight-icons/tests/dummy/public/assets/images/icon-sizes.png)
+
+* Use the `16px` size in product interfaces
+* Use the `24px` size in marketing pages
+
+#### Guidance
+
+:white_check_mark: Display icons at either `16px` or `24px`  
+:white_check_mark: Use `16px` icons by default in product interfaces  
+:white_check_mark: Consider using `24px` icons in product interfaces for empty states  
+:warning: Take care if choosing to display icons at sizes other than `16px` and `24px`  
+_There may be cases where 16px and 24px values don't fit a design - Flight icons can be resized in these cases but be aware that the design is not optimized for values other than these_
+
+#### Updating existing interfaces
+
+:white_check_mark: Where the current icon size is less than or equal to `20px`, replace icons with `16px` versions  
+:white_check_mark: If the size of icon to be replaced is at least `21px`, first try to replace it with a `24px` version  
+_If 24px seems too large, consider dropping down to 16px and reworking the interface for a better fit_
+
+
+### States
+
+Some common state icons that get used within the products include:  
+
+![example of icon states](ember-flight-icons/tests/dummy/public/assets/images/icon-states.png)
+
+### Actions
+
+Some common action icons that get used within the products include:  
+
+![example of common actions](ember-flight-icons/tests/dummy/public/assets/images/icon-actions.png)
+
 ## Contributing
 
 See the [Contributing](CONTRIBUTING.md) guide for details.
 
+### How to contribute a new icon in Figma (HashiCorp staff only)
```

Comment:
Still catching up, looks like this was all brought in as part of https://github.com/hashicorp/flight/pull/157/files except for this section "How to contribute a new icon in Figma (HashiCorp staff only)" do you still want to merge that part in? @heatherlarsen @MelSumner